docs/userguide/labels-custom-metadata.md

```diff
@@ -1,230 +1,115 @@
 <!--[metadata]>
 +++
-title = "Apply custom metadata"
-description = "Learn how to work with custom metadata in Docker, using labels."
-keywords = ["Usage, user guide, labels, metadata, docker, documentation, examples,  annotating"]
+title = "Managing Docker object labels"
+description = "Description of labels, which are used to manage metadata on Docker objects."
+keywords = ["Usage, user guide, labels, metadata, docker, documentation, examples, annotating"]
 [menu.main]
-parent = "engine_guide"
+parent = "engine_guide_intro"
 weight=90
 +++
 <![end-metadata]-->
 
-# Apply custom metadata
+# About labels
 
-You can apply metadata to your images, containers, volumes, networks, nodes, services or daemons via
-labels. Labels serve a wide range of uses, such as adding notes or licensing
-information to an image, or to identify a host.
+_Labels_ are a mechanism for applying metadata to Docker objects, including:
 
-A label is a `<key>` / `<value>` pair. Docker stores the label values as
-*strings*. You can specify multiple labels but each `<key>` must be
-unique or the value will be overwritten. If you specify the same `key` several
-times but with different values, newer labels overwrite previous labels. Docker
-uses the last `key=value` you supply.
+- Images
+- Containers
+- Local daemons
+- Volumes
+- Networks
+- Swarm nodes
+- Swarm services
 
->**Note:** Support for daemon-labels was added in Docker 1.4.1. Labels on
->containers and images were added in Docker 1.6.0
+You can use labels to organize your images, record licensing information, annotate
+relationships between containers, volumes, and networks, or in any way that makes
+sense for your business or application.
 
-## Label keys (namespaces)
+A label is a key-value pair, stored as a string. You can specify multiple labels
+for an object, but each key-value pair must be unique within an object. If the
+same key is given multiple values, the most-recently-written value overwrites
+all previous values.
 
-Docker puts no hard restrictions on the `key` used for a label. However, using
-simple keys can easily lead to conflicts. For example, you have chosen to
-categorize your images by CPU architecture using "architecture" labels in
-your Dockerfiles:
+# Label keys and values
```

Comment:
We could move this heading above the previous paragraph to prevent having two headings without text (think there was a nice writers term for that, but I forgot)